docker/docker-compose.blockchain-link-test.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
```

Comment:
If not using "extends" keyword, I think it is better to go with `version: '3'`

Reply:
Oky, done!